staging/src/k8s.io/code-generator/cmd/client-gen/args/args.go

```diff
@@ -53,13 +56,70 @@ type CustomArgs struct {
 	FakeClient bool
 }
 
-func (ca *CustomArgs) AddFlags(fs *pflag.FlagSet) {
-	pflag.Var(NewGVPackagesValue(&ca.GroupVersionToInputPath, &ca.Groups, nil), "input", "group/versions that client-gen will generate clients for. At most one version per group is allowed. Specified in the format \"group1/version1,group2/version2...\".")
+func NewDefaults() (*args.GeneratorArgs, *CustomArgs) {
+	genericArgs := args.Default().WithoutDefaultFlagParsing()
```

Comment:
Doesn't `WithoutDefaultFlagParsing()` prevent passing `glog` flags?

Reply:
What should it? We add the Go flags (including glog) to the flag set.

Reply:
`--logtostderr` for example. If I make it the default we may want to disable it. The `-v` flag is another example. AFAIK only the standard `flags` package supports these flags, and pflags requires you to explicitly copy the `flags.CommandLine` flagset, hence the need for the default flagset.

Reply:
OK scratch that, there's a call to `AddGoFlagSet()` in `main.go` (y)

Reply:
:)